include/bout/coordinates.hxx

```diff
@@ -53,6 +53,12 @@ public:
   /// Constructor interpolating from another Coordinates object
   Coordinates(Mesh *mesh, const CELL_LOC loc, const Coordinates* coords_in);
   
+  /// Standard constructor from input
```

Comment:
Should we make this private and make google test (or a helper) a friend such that users don't start trying to use this?

Reply:
(or actually test something derived from coordinates that just provides this constructor and no other functionality?)

Reply:
I'd rather not go with the first, just so as to avoid making googletest a hard dependency. The second might be a reasonable approach though.

But do we need to hide it? It's not actively harmful -- one has to not only make a `Coordinates` object but also tell the `Mesh` about it somehow, and we don't currently have anything for that. I had to add `setCoordinates` to `FakeMesh` to actually use this in the tests

Reply:
That's reasonable. I'll try using a derived class in the tests